README

```diff
@@ -1 +1,8 @@
+
+hi
+hellooooooo
+this dnyani chavan
+how r u 
+i am fine
+
```

Comment:
Please remove above and add period to the next line

Reply:
sure